transbank/webpay/webpayplus/transaction.py

```diff
@@ -56,3 +58,37 @@ def create(cls, buy_order, session_id, amount,
             else:
                 raise TransactionCreateException(http_response.status_code, e.args)
 
+    @classmethod
+    def commit(cls, token_ws, options=None):
+        commerce_code = None
+        api_key = None
+        base_url = None
+        if options is None:
+            commerce_code = WebpayPlus.commerce_code()
+            api_key = WebpayPlus.api_key()
+            base_url = WebpayPlus.integration_type_url()
+        else:
+            commerce_code = options.commerce_code
+            api_key = options.api_key
+            WebpayPlus.integration_type_url = options.integration_type
+            base_url = WebpayPlus.integration_type_url()
+
+        headers = {
+            "Tbk-Api-Key-Id": commerce_code,
+            "Tbk-Api-Key-Secret": api_key,
+            "Content-Type": "application/json",
+        }
+
+        http_client = WebpayPlus.http_client
+        final_url = base_url + cls.COMMIT_TRANSACTION_ENDPOINT + "/" + token_ws
+
+        try:
+            http_response = http_client.put(final_url, headers=headers)
+            http_response.raise_for_status()
+            response_json = http_response.json()
+            return TransactionCommitResponse(response_json)
+        except Exception as e:
+            if 'http_response' in locals():
```

Comment:
How Python clean or treat this local variables? What if for example, you run the SDK in a web site and user_1 run in method that declare the variable `http_response`. Is this variable cleaned from locals when method ends? or it is still alive when user_2 run a different method?

Reply:
This local variables are available in the each instance only, they aren't shared with any other instance else. Even sometime ago when I was using/modifiying class attributes, the thing which you are referring it happened

Reply:
Ok